packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -229,6 +233,9 @@ const DiscussionThreadInput = forwardRef((props: Props, ref: any) => {
     CreateTaskMutation(atmosphere, {newTask}, {})
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const addPoll = () => {}
```

Comment:
you can always put an inline comment in the body to get around this lint rule
https://github.com/ParabolInc/parabol/blob/6bf32993d5f6d7386f38c2fc5e7b27609740deea/packages/client/Atmosphere.ts#L59-L62